source/content/drupal-cms.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Drupal CMS on Pantheon
+description: Learn about installing and managing Drupal CMS sites on Pantheon.
+tags: [site, D11]
+permalink: docs/drupal-cms
+contributors: [alexmoreno, stevector]
+contenttype: [doc]
+innav: [true]
+categories: [create]
+cms: [drupal]
+audience: [development]
+product: [dashboard, terminus]
+integration: [--]
+reviewed: "2024-02-10"
+---
+
+# Drupal CMS
+
+Released in January of 2025, Drupal CMS is a relatively new wrapper around Drupal Core.
```

Comment:
Defining Drupal CMS as "relatively new" will date this docs page and require us to come back at some point in the future to adjust the language when it is no longer new. I think we can solve the problem by just removing "relatively new".
```suggestion
Released in January of 2025, Drupal CMS is a wrapper around Drupal Core.
```